go/merkletree/merkle_verifier_test.go

```diff
@@ -19,7 +22,7 @@ func verifierCheck(v *MerkleVerifier, leafIndex, treeSize int64, proof [][]byte,
 		return err
 	}
 	if want := root; !bytes.Equal(got, want) {
-		return fmt.Errorf("got root:\n%v\nexpected:\n%v", got, want)
+		return fmt.Errorf("got root:\n%x\nexpected:\n%x", got, want)
```

Comment:
I'll let it slide this time, but this looks like a lint warning that's been fixed, unrelated to this PR... :wink: